crates/wasmparser/src/validator/operators.rs

```diff
@@ -850,29 +869,28 @@ impl OperatorValidator {
                 self.pop_operand(None, resources)?;
             }
             Operator::Select => {
+                fn is_num_or_vec(ty: ValType) -> bool {
+                    matches!(
+                        ty,
+                        ValType::I32
+                            | ValType::I64
+                            | ValType::F32
+                            | ValType::F64
+                            | ValType::V128
+                            | ValType::Bot
+                    )
+                }
```

Comment:
Good catch, this will merge conflict but that's fine.   As I'm sure you saw, the 2.0 spec has this definition, but not function-references, which is why we accidentally reverted this

Reply:
Yep! Should be a simple conflict to resolve, though.